.travis.yml

```diff
@@ -30,11 +28,8 @@ script:
   - qiita_env make --no-load-ontologies
   - if [ ${TEST_ADD_STUDIES} == "True" ]; then test_data_studies/commands.sh ; fi
   - if [ ${TEST_ADD_STUDIES} == "False" ]; then qiita_test_install ; fi
-  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests --with-doctest --with-coverage ; fi
+  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests -v --with-doctest --with-coverage ; fi
```

Comment:
Wait, now #1008 failed and it is showing the exact same behavior as
this, I'm uncertain of the cause for this problem. Will investigate
further as soon as I can.

On (Mar-21-15| 7:09), Antonio Gonzalez wrote:
> > @@ -30,11 +28,8 @@ script:
> >    - qiita_env make --no-load-ontologies
> >    - if [ ${TEST_ADD_STUDIES} == "True" ]; then test_data_studies/commands.sh ; fi
> >    - if [ ${TEST_ADD_STUDIES} == "False" ]; then qiita_test_install ; fi
> > -  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests --with-doctest --with-coverage ; fi
> > +  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests -v --with-doctest --with-coverage ; fi
> 
> Saw your last commit message, any idea why? Note that IMOO the -v makes things less clear ... but I guess we can get used to the new output format 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1011/files#r26892097